kinder/pkg/cluster/manager/actions/kubeadm-config.go

```diff
@@ -272,14 +325,101 @@ func getKubeadmConfig(c *status.Cluster, n *status.Node, data kubeadm.ConfigData
 	// if the node is the bootstrap control plane, then all the objects used as init time
 	if n == c.BootstrapControlPlane() {
 		return selectYamlFramentByKind(patched,
+			"MasterConfiguration", //v1alpha2
```

Comment:
v1alpha2 was removed in 1.13, while v1alpha3 was already available in 1.12.
1.12 is outside of the skew but we must support it for upgrades.
i think we can avoid the v1alpha2 MasterConfiguration and NodeConfiguration objects and use v1alpha3, but i may be missing something?

Reply:
you are right, we can nuke v1alpha2, but this is out of the scope of this PR